pig/src/main/java/org/jboss/pnc/bacon/pig/impl/repo/ExternalArtifactDownloader.java

```diff
@@ -54,7 +54,7 @@ public static File downloadExternalArtifact(GAV gav, File targetPath, boolean so
             FileDownloadUtils.downloadTo(downloadUrl, targetPath);
         } catch (RuntimeException any) {
             if (sourcesOptional && "sources".equals(gav.getClassifier()) || "javadoc".equals(gav.getClassifier())) {
-                log.warn("Unable to download sources for {}", gav, any);
+                log.warn("Unable to download sources for {}: {}", gav, any.getMessage());
```

Comment:
just spoke about this with Nick, with your change output would be smaller so I would leave it for now, but I don't know how many times it could be printed to user and if it's that important so :see_no_evil: . At the end I would see bacon informing users just about major steps in process and errors. If users want detailed logs they could switch to verbose output

Reply:
Yeah we can adjust later in another commit. We'll learn as we go! :)